demos/frontend-wt/.gitignore

```diff
@@ -0,0 +1,265 @@
+## Ignore Visual Studio temporary files, build results, and
+## files generated by popular Visual Studio add-ons.
+
+# User-specific files
+*.suo
+*.user
+*.userosscache
```

Comment:
We should favor out-source-builds.

Reply:
All these files were automatically generated by msvs